config/docs-json.config.js

```diff
@@ -46,7 +47,52 @@ StyleDictionary.registerFormat({
  *   comment: '...'
  * },
  */
-function getDocsShape() {
+
+ var levels = {
+	"fail": {
+		range: [0, 3],
+		color: "hsl(0, 100%, 40%)"
+	},
+	"aa-large": {
+		range: [3, 4.5],
+		color: "hsl(40, 100%, 45%)"
+	},
+	"aa": {
+		range: [4.5, 7],
+		color: "hsl(80, 60%, 45%)"
+	},
+	"aaa": {
+		range: [7, 22],
+		color: "hsl(95, 60%, 41%)"
+	}
+};
+
+function getLevel(ratio) {
+  for (const [key, value] of Object.entries(levels)) {
+    if (ratio > value.range[0] && ratio < value.range[1]) {
+      return key
+    }
+  }  
+}
+
+function getContrastCheck(value, element, allTokens) {
+  const format = element.replaceAll('.', '-').replace('&', 'and')
+  const withPrefix = 'telekom-' + format
+  const currentToken = allTokens.find(el => el.name === withPrefix)
```

Comment:
It's probably safer to leave `element` untouched and compare against the same dot separated path?

```js
const currentToken = allTokens.find(token => token.path.join('.') === element)
```

(Also would rename `element` to `path` or `targetPath`)

Reply:
also related to https://github.com/telekom/design-tokens/pull/143#discussion_r804585845 — I think it's safer if we avoid the string manipulation and compare the stringified paths…